src/styled/Common.ts

```diff
@@ -321,6 +321,27 @@ export const FlexContainer = styled.div<
     ${Breakpoint}
     ${Animations}
 `;
+export const GradientBoxShadow = styled.div<
```

Comment:
Now that i merged in some changes, this can be accomplished in the new format if you look at Containers.ts, something like this: 
```
export const GradientBoxShadow = styled(Container)`
    box-shadow: var(--gradient-box-shadow);
`;
```

Reply:
And you'll have to bring that into the Common/index file as well if it's to be a Common component.